ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh

```
@@ -4,6 +4,18 @@ set -o nounset
 
+exec > >(tee -i "${SHARED_DIR}/setup-output.log") 2>&1
+
+trap 'rc=$?; if [[ $rc -ne 0 ]]; then
+    cat > "${SHARED_DIR}/diagnose-prompt.txt" << PROMPT
+The standard Multi-Node OpenShift (MNO) baremetal cluster installation failed.
+The cluster was being installed using kcli and ansible playbooks on baremetal nodes.
+Analyze the setup log below to identify the root cause of the installation failure.
+Look for ansible task failures, network connectivity issues, node provisioning errors,
+or OpenShift installer errors. Provide a summary of what went wrong and suggest fixes.
+PROMPT
+fi' EXIT
```

Shellcheck (0.11.0)

[warning] rc is referenced but not assigned.

(SC2154)